### PR TITLE
create Footer component

### PR DESCRIPTION
**Pull request by Vlad-coder-678**

### CODE REVIEW

**Comment by AnAtoliy-AA** [comment]
Every new branch You should create from develop branch

**Comment by Evgenia-cyber** [comment]
You can delete logo.svg from src

File: poker/.prettierrc.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
-    trailingComma: "es5",
```
Comment (GoodFellow23) [inline poker/.prettierrc.js:2]: this chenges are here https://github.com/GoodFellow23/SilentTeam-PP-FE/pull/3

File: poker/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (GoodFellow23) [inline poker/.eslintrc:1]: the same https://github.com/GoodFellow23/SilentTeam-PP-FE/pull/4/files#r702038984

File: poker/.babelrc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (GoodFellow23) [inline poker/.babelrc:1]: the same https://github.com/GoodFellow23/SilentTeam-PP-FE/pull/4/files#r702038984

File: poker/package.json [hunk 4]
```diff
@@ -42,18 +42,24 @@
     ]
   },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.15.4",
```
Comment (GoodFellow23) [inline poker/package.json:45]: the same

File: poker/public/index.html [hunk 5]
```diff
@@ -3,41 +3,23 @@
...
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
```
Comment (GoodFellow23) [inline poker/public/index.html:13]: the same

File: poker/src/App.jsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import './App.css';
```
Comment (GoodFellow23) [inline poker/src/App.jsx:1]: will be conflict with https://github.com/GoodFellow23/SilentTeam-PP-FE/pull/3

**Pull request merged**